sdk/metricsadvisor/Azure.AI.MetricsAdvisor/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 ### Breaking Changes
 - Renamed all `SkipCount` listing options to `Skip`. Affected classes include `GetAlertsOptions`, `GetDataFeedsOptions`, `GetHooksOptions`, and others.
 - Renamed all `TopCount` listing options to `MaxPageSize`. Affected classes include `GetAlertsOptions`, `GetDataFeedsOptions`, `GetHooksOptions`, and others.
+- Removed data feed sources `ElasticsearchDataFeedSource` and `HttpRequestDataFeedSource`.
```

Comment:
should you clarify the why? or is there another thing the user could use instead of this? 
Just wondering if I was a user of this library and my code had this and then suddenly it was deleted.. not sure what to do...

Reply:
It won't be supported by the service anymore because of security concerns, but we're not aware of the specifics. According to the service team, they know exactly how many and what customers are using the removed features, and they intend to notify them.

I can update it to
> Removed data feed sources `ElasticsearchDataFeedSource` and `HttpRequestDataFeedSource` as they are not supported by the service anymore. A different type of data feed source must be used for data ingestion instead.